vllm/entrypoints/openai/cli_args.py

```diff
@@ -246,8 +246,7 @@ def make_arg_parser(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                         type=int,
                         default=None,
                         help='Max number of prompt characters or prompt '
-                        'ID numbers being printed in log.'
-                        '\n\nDefault: Unlimited')
```

Comment:
I think we should say None means Unlimited.

Reply:
added